### PR TITLE
Add tests, Travis, etc.

### PR DESCRIPTION
**Pull request by tavinathanson**
In this PR:
- Tests for basic functionality and for counting variants.
- Some minor refactoring upon noticing issues while writing tests.
- Travis, Coveralls, and linting.

I'll move this repo to `hammerlab` after merging this PR.

TODO:
- I need to update our internal repo to use the refactoring.
- I'll update Travis/Coveralls badges to `hammerlab` after moving.
- All add automatic `PyPi` deploy after moving.

@arahuja

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+pandas>=0.15
+seaborn>=0.7.0
+scipy>=0.16.1
```
Comment (arahuja) [inline requirements.txt]: Do we want to enforce scipy 0.17 for the mann whitney issues?

Comment (tavinathanson) [inline requirements.txt]: scipy 0.17.1 will likely revert back the behavior (hoping to submit a scipy PR for that soon), so I think we're better off explicitly specifying two-sided. I figured I'd do that in a separate PR.

Comment (arahuja) [inline requirements.txt]: Yes, I agree, but I don't think `0.16.1` can?

Comment (tavinathanson) [inline requirements.txt]: Yeah, fair enough. Will add `0.17` here 👍

**Comment by tavinathanson** [comment]
@arahuja Cool if I merge today?

**Comment by arahuja** [comment]
@tavinathanson Sure, looks good!
